src/parser/name_resolution.c

```diff
@@ -2726,7 +2726,7 @@ pt_bind_names (PARSER_CONTEXT * parser, PT_NODE * node, void *arg, int *continue
 		  prev->next = next;
 		}
 	      cnf->next = NULL;	/* cut-off link */
-
+	      PT_EXPR_INFO_CLEAR_FLAG (cnf, PT_EXPR_INFO_LEFT_OUTER | PT_EXPR_INFO_RIGHT_OUTER);
```

Comment:
빈 줄로 구분을 해줬으면 좋겠습니다.
```suggestion

	      PT_EXPR_INFO_CLEAR_FLAG (cnf, PT_EXPR_INFO_LEFT_OUTER | PT_EXPR_INFO_RIGHT_OUTER);

```